src/hooks/useFhirClient.tsx

```diff
@@ -40,8 +44,14 @@ export function useFhirClient() {
     // TODO: add code, date, & other query param capabilities
     // TODO: consider using fhir-search across the board (documenting delay)
 
-    return useQuery(
-      [`search-${resourceType}`, params],
+    const fetchNextPage = () => {
+      if (hasMoreData) {
+        setNext((prev) => prev + 10);
```

Comment:
I just realized `pageSize` is an optional parameter. I'll update this to increment `next` by `pageSize`.